src/Illuminate/Contracts/Validation/Validator.php

```diff
@@ -13,6 +13,13 @@
      */
     public function validate();
 
+    /**
+     * Return validated value.
```

Comment:
Change to: "Return validated values." it should be plural

Reply:
@jmarcher Thanks for review. Fixed.